src/assets/ElmAsset.js

```diff
@@ -0,0 +1,43 @@
+const JSAsset = require('./JSAsset');
+const localRequire = require('../utils/localRequire');
+const process = require('process');
+
+// Currently no pretty way to install Elm tools on behalf of the user :/
+async function compile(name, options) {
+  try {
+    const elmCompiler = await localRequire('node-elm-compiler', name);
+    // compileToStringSync does not throw an Error, even when there is one.
+    // Other methods will throw an error, but do not bubble up to Parcel
+    const data = elmCompiler.compileToStringSync(name, options);
+    if (!data) {
+      throw 'You either have an error in your code, or are missing the Elm tools here https://guide.elm-lang.org/install.html.';
+    }
+    return data;
+  } catch (error) {
+    throw error;
+  }
+}
+
+class ElmAsset extends JSAsset {
+  async getParserOptions() {
+    const defaultOptions = {
+      yes: true,
```

Comment:
is this really an option?

Reply:
`yes` option is to reply "yes" to all automated prompts.
Here are the full list of options available for node-elm-compiler.
```
var defaultOptions  = {
  emitWarning: console.warn,
  spawn:      spawn,
  cwd:        undefined,
  pathToMake: undefined,
  yes:        undefined,
  help:       undefined,
  output:     undefined,
  report:     undefined,
  warn:       undefined,
  debug:      undefined,
  verbose:    false,
  processOpts: undefined,
};
```
Is there any way to customize these options?

Reply:
@rtorr I've created a pull request on your master regarding to adding user options to `node-elm-compiler` https://github.com/rtorr/parcel/pull/1

Reply:
accepted